app/src/main/java/com/example/chat/MessageRecyclerAdapter.java

```diff
@@ -80,22 +80,22 @@ public void changeCursor(Cursor cursor)
         notifyDataSetChanged();
     }
 
-    public void addRow(message m)
-    {
-        MatrixCursor matrixCursor = new MatrixCursor(new String[] { messageEntry.COLUMN_RECEIVED, messageEntry.COLUMN_TEXT,messageEntry.COLUMN_SEND_TIME });
-        if(m.received)
-        {
-            matrixCursor.addRow(new Object[]{1, m.text, m.send_time});
-        }
-        else
-        {
-            matrixCursor.addRow(new Object[]{0, m.text, m.send_time});
-        }
-        mCursor = new MergeCursor(new Cursor[] {  mCursor,matrixCursor });
-
-        populateColumnPositions();
-        this.notifyDataSetChanged();
-    }
+//    public void addRow(Message m)
+//    {
+//        MatrixCursor matrixCursor = new MatrixCursor(new String[] { messageEntry.COLUMN_RECEIVED, messageEntry.COLUMN_TEXT,messageEntry.COLUMN_SEND_TIME });
+//        if(m.received)
+//        {
+//            matrixCursor.addRow(new Object[]{1, m.text, m.send_time});
+//        }
+//        else
+//        {
+//            matrixCursor.addRow(new Object[]{0, m.text, m.send_time});
+//        }
+//        mCursor = new MergeCursor(new Cursor[] {  mCursor,matrixCursor });
+//
+//        populateColumnPositions();
+//        this.notifyDataSetChanged();
+//    }
```

Comment:
Please remove commented out code.

Reply:
Okay, sure I'll be removing all commented out code in the next commit.